client/src/Host.jsx

```diff
@@ -1,11 +1,15 @@
 import React from 'react';
+import {HostContainer, HostName} from './styles.js'
 
 const Host = (props) => {
   return (
-    <div>
-      <img src={props.site.hostAvatar}></img>
-      Hosted By {props.site.hostName}
-    </div>
+    <HostContainer>
+      <img style={{height: '98px', width: '98px', borderRadius: '50%'}}src={props.site.hostAvatar}></img>
```

Comment:
consider separating this style out instead of doing it inline

Reply:
Yeah that's something that I thought about as well. I think I left it in just because it was the only instance of that image, but I will go back and change it for cleaner code!